webpack.icons.config.js

```diff
@@ -10,12 +10,10 @@ const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 module.exports = {
   mode: 'development',
   entry: {
-    'clr-icons-all': './src/clr-icons/autoload-all.umd.ts',
-    'clr-icons-all.min': './src/clr-icons/autoload-all.umd.ts',
-    'clr-icons': './src/clr-icons/api-autoload.umd.ts',
-    'clr-icons.min': './src/clr-icons/api-autoload.umd.ts',
-    'shapes/all': './src/clr-icons/shapes/all.umd.ts',
-    'shapes/all.min': './src/clr-icons/shapes/all.umd.ts',
+    'clr-icons': './src/clr-icons/api.umd.ts',
```

Comment:
Do we need these non-minified bundles `clr-icons` and `clr-icons-all`?

Reply:
It can be useful for someone trying to read the source code, and it is pretty common to ship both a min and non-min version. If we didn't ship these as min, then we shouldn't ship any of the shapes either.